README.md

```diff
@@ -72,6 +74,7 @@ The `tabs` field is an array of objects with the following fields:
 | `url`               | String (Required)           | Starting page for this tab                                             |
 | `customJs`          | Array of Strings (Optional) | Custom JS URLs (see [Custom JS/CSS](#custom-jscss))                    |
 | `customCss`         | Array of Strings (Optional) | Custom CSS URLs (see [Custom JS/CSS](#custom-jscss))                   |
+| `customCookie`      | Array of Cookies (Optional) | Custom Cookies injection (see [Custom Cookies](#custom-cookies))       |
```

Comment:
Can we call the field `customCookies` instead (plural)?

Also, I think it needs its own documentation table, like we have for `tabs`. I'd prefer if users didn't need to read WebKit docs to understand Multi settings.

Reply:
Done

Reply:
The documention is more complex than we can simply hold in a table as there some dependency between some of the parameters.
One example is `version` and `discard`

If `version` is `0` (the default) then the default for `discard` is `FALSE` and otherwise it's `TRUE`

how do you want to approach this?

Reply:
Good point, I left the link. On further reflection, I think this feature will be rarely used, so the entire table seemed like overkill.